apps/galaxy/src/services/signup/useSignUp.ts

```diff
@@ -84,5 +84,27 @@ export default function useSignUp() {
     }
     setIsLoading(false);
   };
-  return { signUp, isLoading };
+
+  const checkEmailIsExists = async (email: string | null | undefined) => {
+    try {
+      const response = await axios.get(`/staff/emails/${email}`);
+      if (response.status === 200) {
+        return false;
+      }
+    } catch (error) {
+      const err = error as AxiosError;
```

Comment:
I think since [this comment](https://github.com/courtcanva/cc-multiverse/pull/10#discussion_r1001899535), we've decided to run this logic via `if (error instanceof AxiosError)` instead of casting it to `AxiosError`. Better update this.

Reply:
Good.